### PR TITLE
Change warning ignores to use a string identifier

### PR DESCRIPTION
**Pull request by Scimon**
Tested in Godot 4 rc1.

### CODE REVIEW

File: audio/bpm_sync/bpm_sync.gd [hunk 1]
```diff
@@ -42,7 +42,7 @@ func _process(_delta):
 	var beat = int(time * BPM / 60.0)
 	var seconds = int(time)
 	var seconds_total = int($Player.stream.get_length())
-	@warning_ignore(integer_division)
+	@warning_ignore('integer_division')
```
Comment (aaronfranke) [inline audio/bpm_sync/bpm_sync.gd:45]:
```suggestion
	@warning_ignore("integer_division")
```

**Review by Calinou — CHANGES_REQUESTED** [review]
There are several warning ignores not using quotes:

```
❯ rg -i "warning_ignore"
audio/spectrum/show_spectrum.gd
14:	@warning_ignore(integer_division)

3d/voxel/settings.gd
10:@warning_ignore(unused_private_class_variable)

audio/bpm_sync/bpm_sync.gd
45:	@warning_ignore(integer_division)
```

Please also change them in this pull request.

Also, is this change recommended by the GDScript style guide, or was this a recent breaking change in GDScript?

**Comment by Scimon** [comment]
I opened the demo up in 4.0rc1 and it crashed due to that error so I figured I'd fix it. 
I didn't look for that use of warning ignore in other location because I was planning on doing demos on a case by case basis.

**Comment by Scimon** [comment]
Sorry about the single quotes day job is Perl and general style is to use single quotes unless you need string interpolation.

**Review by aaronfranke — APPROVED** [review]

**Comment by aaronfranke** [comment]
> or was this a recent breaking change in GDScript?

This. See #838 and https://github.com/godotengine/godot/issues/72218

**Review by Calinou — APPROVED** [review]
=
Thanks! Congratulations for your first merged pull request :tada:

**Comment by Scimon** [comment]
I'll try and work through some more demo's at the weekend. :)

**Pull request merged**